tensor2tensor/data_generators/text_encoder.py

```diff
@@ -36,6 +37,10 @@
 EOS = '<EOS>'
 RESERVED_TOKENS = [PAD, EOS]
 
+# Character that will be used instead of those with
+# ordinal numbers > 255
+UNK_CHAR = '×' # ordinal 215
```

Comment:
@rsepassi However, there is another possible approach. Python 2 encodes its strings into bytes using UTF-8, so for instance (using some characters from Icelandic, my native language):
```
>>> [ ord(c) for c in "þæðö" ]
[195, 190, 195, 166, 195, 176, 195, 182]
```
In Python3 we can get the same result by explicitly encoding to UTF-8:
```
>>> [ c for c in "þæðö".encode("utf-8") ]
[195, 190, 195, 166, 195, 176, 195, 182]
```
I will modify the pull request to use this approach, which has the advantage of making the Python3 version fully compatible with the Python2 version.